iceoryx_dds/source/iceoryx_dds/dds/cyclone_data_reader.cpp

```diff
@@ -45,13 +45,23 @@ void iox::dds::CycloneDataReader::connect() noexcept
         auto subscriber = ::dds::sub::Subscriber(CycloneContext::getParticipant());
 
         auto qos = ::dds::sub::qos::DataReaderQos();
+
+        /// Is required for the Gateway. When two iceoryx publisher are publishing on the same
```

Comment:
This documentation block is a bit confusing to me. What exactly is required for the gateway?
I guess not  everybody will understand what a `iceoryx remove instance` is.

Reply:
This is a typo, I meant to write remote iceoryx instance.